js/models/questionModel.js

```
@@ -406,6 +408,32 @@ class QuestionModel extends ComponentModel {
     return {};
   }
 
+  addContentObjectContextActivities() {
+    // SCORM doesn't necessarily need course context as implied in reports (exclude via spoor)
+    this.getAncestorModels()
+      .filter(model => model.isTypeGroup('contentobject'))
+      .forEach(model => {
+        const id = model.get('_id');
+        const type = model.get('_type');
+        const title = model.get('title') || model.get('displayTitle');
+        this.addContextActivity(id, type, title);
+      });
+  }
+
+  addContextActivity(id, type, title) {
+    const isIncluded = this._contextActivities.some(activity => activity.id === id);
+    if (isIncluded) return;
```

A second call with the same `id` may be deliberate. Fail silently or overwrite?